website/docs/docs/core/connect-data-platform/athena-setup.md

```diff
@@ -4,7 +4,7 @@ description: "Read this guide to learn about the Athena warehouse setup in dbt."
 meta:
   maintained_by: dbt Labs
   authors: dbt Labs
-  github_repo: 'dbt-labs/dbt-athena'
+  github_repo: 'dbt-labs/dbt-adapters'
```

Comment:
hey @colin-rogers-dbt - would we want to link directly to `/dbt-labs/dbt-adapters/tree/main/dbt-athena` instead of the main dbt-adapters page?

Reply:
also we should probably update the links to all the first party adapters - happy to do that on my end!

Reply:
yea we probably should link to the sub-directory `/dbt-labs/dbt-adapters/tree/main/dbt-athena`

Reply:
great stuff, thank you! i'll update and merge! i'll create a separate pr for the other first party adapters too. great shout @colin-rogers-dbt !

Reply:
argh it's giving a 404 badge issue so going to keep it to adapters
![Screenshot 2025-03-11 at 17 16 18](https://github.com/user-attachments/assets/0fd15b31-222d-43b3-b218-407543fa32f3)